cmd/webhook/admission/admission.go

```diff
@@ -83,8 +83,8 @@ func (RouteGroupAdmitter) admit(req *admissionRequest) (*admissionResponse, erro
 			},
 		}, nil
 	}
-
-	err = definitions.ValidateRouteGroup(&rgItem)
+	rgValidator := definitions.RouteGroupValidator{}
+	err = rgValidator.Validate(&rgItem)
```

Comment:
Lets have a default global `var defaultRouteGroupValidator definitions.RouteGroupValidator` and make existing function `definitions.ValidateRouteGroup` call `defaultRouteGroupValidator.Validate` for now.